doc/api/fs.md

```diff
@@ -1210,6 +1210,7 @@ changes:
   * `start` {integer}
   * `end` {integer}
   * `highWaterMark` {integer}
+* Returns: {ReadStream}
```

Comment:
Should be `{stream.Readable}`
See [type-parser.js#L81-L84](https://github.com/nodejs/node/blob/0e7b61229aa602e55c5fb034a63d7da97eecff3b/tools/doc/type-parser.js#L81-L84)

Reply:
Thank you for your kind pointing out!